workflow/envs/excel.yaml

```diff
@@ -2,5 +2,5 @@ channels:
   - conda-forge
   - bioconda
 dependencies:
-  - openpyxl =3.0.7
-  - pandas =1.2.5
+  - openpyxl =3.1
+  - pandas =1.5
```

Comment:
It actually seems like this `excel.yaml` file is not used anywhere, so we should probably simply remove it, right?

Reply:
But the respective rule to generate excel files does not exist any more in the current state of this branch. Also, I am doing a sweep and removing any `envs/*.yaml` files that are never used... :grimacing: